components/docs-chef-io/content/automate/servicenow_integration.md

```diff
@@ -0,0 +1,336 @@
+
+The Chef Automate application exposes the REST API endpoint that facilitates the communication between Chef Automate and the ServiceNow instance.
+
+- Visit the ServiceNow store at <https://store.servicenow.com>.
+- Get the **Chef Automate Integration App** application.
+- From the ServiceNow instance, navigate to the **System Applications** > **Applications** menu.
+- From the **Downloads** tab, install the **Chef Automate** application.
+
+### Creating Application Users
+
+The Chef Automate application provides several roles appropriate for integration. The existing or new ServiceNow users can be assigned these roles, which are as follows:
+
+- x_chef_automate.admin
+- x_chef_automate.user
+- x_chef_automate.api
+
+These roles are part of the package. Users can create their requisite roles and controls if any further restrictions are needed.
+
+#### Role x_chef_automate.admin
+
+The `x_chef_automate.admin` role is assigned to a user other than a System Administrator to allow administration of the application properties and logs. Thus, a user who is not a system administrator performs the administration. 
+
+Note that a System Administrator can perform all tasks that this role can. In addition, the **Admin** role is added by default in ServiceNow.
+
+The **Admin** role grants user access to the:
+
+- Asset Imports module
+- Properties
+- Automate instances
+- Servers
+- Chef infra client runs
+- Compliance reports profiles
+- Compliance report results
+- Compliance Profiles
+- Compliance Profiles results
+- Support
+- Events 
+- Script includes
+- Transform maps
+- Chef servers
+
+#### Role x_chef_automate.user
+
+The `x_chef_automate.user` role is suitable for those users who require application access without administration rights. The role grants a user access to the:
+
+- Chef Automate menu
+- Servers module
+- Client runs module
+- CI cookbooks module
+- Compliance reports module
+- Compliance reports profiles module
+- Compliance reports module
+- Compliance profiles module
+- Compliance profiles controls module
+- Logs module
+- Properties module
+- Support module
+
+Ensure that you need to assign the _OOB ITIL_ role for integration with CMDB data in ServiceNow.
+
+#### Role x_chef_automate.api
+
+The `x_chef_automate.api` role is suitable for users responsible for integrating the Chef Automate data into the application. It is advisable to create a new user specifically for this role. This user's credentials are needed to configure the Chef Automate server for communication with the application. 
+
+Note that for integration with CMDB data, you need to assign the _OOB ITIL_ role. In addition, you must select **Web service** access only for this user role.
+
+### References
+
+- [Creating Users in ServiceNow] (https://docs.servicenow.com/csh?topicname=t_CreateAUser.html)
+- [Assigning Roles in ServiceNow] (https://docs.servicenow.com/csh?topicname=t_AssignARoleToAUser.html)
+
+### Configuring Application Properties
```

Comment:
Should we move it to the Configuration Section below, after the Chef Automate Integration settings are done.

Reply:
sure, make sense